superset-frontend/cypress-base/cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://localhost:8081",
+  "baseUrl": "http://localhost:8088",
```

Comment:
just want to point out that this change could potentially break some user's workflow, but it could still be an improvement.